code/mob/living/critter/skeleton.dm

```diff
@@ -30,18 +30,43 @@
 	name = "skeleton"
 	real_name = "skeleton"
 	desc = "Clak clak, motherfucker."
-	density = 1
```

Comment:
Ok I assume this is handled in the lifeprocesses or something but just to check again: this is intentional right

Reply:
all critters default to having density so theres not much reason to have it unless you are making it false

Reply:
ah I see